msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -89,6 +92,21 @@ void acquireToken(@NonNull final Activity activity,
     @WorkerThread
     IAuthenticationResult acquireTokenSilent(@NonNull final AcquireTokenSilentParameters acquireTokenSilentParameters) throws InterruptedException, MsalException;
 
+//    /**
+//     * Perform the Device Code Flow (DCF) protocol to allow a device without input capability to authenticate and get a new access token.
+//     * This flow is now supported in Broker as well. It also supports requesting Claims using the "claims" Request. Parameter.
+//     *
+//     * @param scopes   the desired access scopes
+//     * @param callback callback object used to communicate with the API throughout the protocol
+//     * @param claims claims Authentication Request parameter requests that specific Claims be returned from the UserInfo Endpoint and/or in the ID Token.
+//     * @param correlationId  correlation id of this request
+//     *
+//     * Important: Use of this API requires setting the minimum_required_broker_protocol_version to
+//     * "13.0" or higher.
```

Comment:
we need to ensure that Teams devices sends this min required version. Did you create a task for sending this?

Reply:
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/3055116